egui_plot/src/lib.rs

```diff
@@ -1517,7 +1531,7 @@ impl<'a> PreparedPlot<'a> {
         };
 
         // Draw cursors
-        let line_color = rulers_color(ui);
+        let line_color = self.ruler_color.unwrap_or_else(|| rulers_color(ui));
```

Comment:
What you call "ruler" is called "cursor" elsewhere

Reply:
the original function is called `rulers_color`, that's why I chose that name 😅
I'll change it 👍